bmc-acf/acf_manager.cpp

```diff
@@ -394,7 +397,21 @@ acf_info ACFCertMgr::installACF(std::vector<uint8_t> accessControlFile)
         }
         else
         {
-            log<level::ERR>("cannot read production key file");
+            log<level::ERR>("cannot read production backup key file");
+            elog<InternalFailure>();
+        }
+    }
+    if (prodBackup2KeyExists && sRc != CeLogin::CeLoginRc::Success)
```

Comment:
nit: best to use () to explicitly show logic precedence for review ease and to avoid dangerous bugs.

Reply:
Added parens